src/main/java/io/quarkus/search/app/SearchService.java

```diff
@@ -58,15 +66,15 @@ public SearchResult<GuideSearchHit> search(@RestQuery @DefaultValue(QuarkusVersi
 
                     if (q != null && !q.isBlank()) {
                         root.add(f.bool().must(f.simpleQueryString()
-                                .field("title").boost(10.0f)
+                                .field("title_%s".formatted(language)).boost(10.0f)
```

Comment:
Maybe declare a util somewhere to handle this field name generation... I.e. in `I18nData`?